servers/rendering_server.cpp

```diff
@@ -2067,6 +2067,16 @@ void RenderingServer::_particles_set_trail_bind_poses(RID p_particles, const Typ
 	particles_set_trail_bind_poses(p_particles, tbposes);
 }
 
+String RenderingServer::get_current_rendering_driver_name() const {
+	// Needs to remain in OS, since it's actually OS that interacts with it, but it's better exposed here.
+	return ::OS::get_singleton()->get_current_rendering_driver_name();
+}
```

Comment:
I would still challenge this comment. In what way does `OS` interact with this? It just seems like the value was parked there for ease of access, but it could as well be in `RenderingServer` a priori. It's mostly set from `DisplayServer` implementations which should have access to both singletons.

I'll have a look to confirm.

Reply:
I've confirmed these can be moved to `RenderingServer` without problems with https://github.com/akien-mga/godot/commit/5ea9478749b8a77ce6e5042cb914cb88e6ff437e.
So I think we should do it properly (amending in my commit should do the trick).

And also moved `OS::_is_gles_over_gl` that similarly doesn't need to be in `OS`, and removed the getter for `_display_driver_id`, was never used since it was added in 2020.

Reply:
Actually some CI tests are failing with my branch. I'll look into it further.